dashboard/client/api/kube-api.ts

```diff
@@ -40,25 +41,67 @@ export interface IKubeApiLinkBase extends IKubeApiLinkRef {
 }
 
 export class KubeApi<T extends KubeObject = any> {
-  static parseApi(apiPath = ""): IKubeApiLinkBase {
+  static parseApi(apiPath = "") {
```

Comment:
Why return type has been removed?

Reply:
Thanks for catching this. I didn't mean to.